TypeScript.YAML-tmLanguage

```diff
@@ -495,16 +675,16 @@ repository:
     - include: '#expression'
 
   expression-operator:
-    name: keyword.others.ts
-    match: \b(delete|in|instanceof|new|typeof|as|is|of)\b
+    name: keyword.operator.ts
```

Comment:
that was a while ago.
i think that i did it to color those keyword in a different color.